corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -524,6 +528,10 @@
         {% include 'hqwebapp/partials/ko_search_box.html' %}
         {% include 'hqwebapp/partials/ko_select_toggle.html' %}
 
+        {% if show_overdue_invoice_modal %}
```

Comment:
This should also check `and not requirejs_main` - and you should test on both a requirejs page and a non-requirejs page if you haven't.

Reply:
@orangejenny can you elaborate a bit on this check? why its needed and how does it effect the working of this script?

Reply:
In a requirejs environment, we don't use script tags. All of the relevant js is fetched by requirejs based on the `requirejs_main` module's dependencies (and their dependencies, and so forth). In this PR, requirejs pages get access to the new js code via the new dependency in `hqwebapp/js/base_main.js`.

If you include a script tag on a requirejs page, it'll likely throw errors because it won't have access to everything it needs. Plus it should already be included in a requirejs way, so even if it works, it'll be running twice.

Reply:
Updated: 69ec2401c7b986ce0b5ee9809fe8cfb5664d74c8